src/corelibs/U2Formats/src/StockholmFormat.cpp

```diff
@@ -94,491 +75,331 @@ enum AnnotationTag {
     NC,
     TC
 };
-// other types not supported
+
+/** Set of supported annotation types. */
 enum AnnotationType {
     FILE_ANNOTATION,
     COLUMN_ANNOTATION,
     UNI_ANNOTATION
 };
 
-struct Annotation {
-    AnnotationType type;
-    AnnotationTag tag;
-    QString val;
-    Annotation(AnnotationType ty, AnnotationTag t, QString v) {
-        type = ty, tag = t;
-        val = v;
-    }
-    virtual ~Annotation() {
-    }
-};
-//#=GF annotations
-struct FileAnnotation : public Annotation {
-    FileAnnotation(AnnotationTag t, QString v)
-        : Annotation(FILE_ANNOTATION, t, v) {
+/** Internal representation of the annotation used internally during parsing. */
+class StockholmAnnotation {
+public:
```

Comment:
If it's fully-public class, so, it's deffinetely a structure. I would suggest use this type in this case

Reply:
Fixed: made it a struct.